### DISCUSSION TITLE
Discord community?

### DISCUSSION THREAD

**Discussion by wisewtf** [getmango/Mango#202] [General]
Would be pretty cool to integrate a nice Discord server for people to meet up and talk about Mango. Don't know if it already exists, I couldn't find it!

**Comment by hkalexling** [comment]
Hey, thanks for reaching out! 

We already have a Gitter room for general discussions, but I do understand that some of you would prefer Discord so I created a new server here: http://discord.com/invite/ezKtacCp9Q. I will put the link in the project README when making a new release.
